pkg/server/app/app.go

```diff
@@ -117,7 +117,7 @@ func (ops *AppOperations) Create(user *database.User, app *App) (Err error) {
 		return auth.ErrPermissionDenied
 	}
 
-	if ops.kops.IngressEnabled() && app.VirtualHost == "" && IsWebApp(app.ProcessType) {
+	if ops.kops.IngressEnabled() && app.VirtualHost == "" && !app.ReserveStaticIp && IsWebApp(app.ProcessType) {
 		return ErrMissingVirtualHost
```

Comment:
What about creating a method called `CheckVirtualHostIsMissing`?

Reply:
@drgarcia1986 should I put this method on `app.go`, using `AppOperations` as its receiver?

Reply:
make senses but only on concrete app implementation.

Reply:
@drgarcia1986 I didn't quite understand what you mean by the concrete app implementation, but I added the `CheckVirtualHostIsMissing` at https://github.com/luizalabs/teresa/pull/599/commits/e48719479e895e30bb9dfbe85f2f552223f3910a